### ISSUE TITLE
重启容器后web配置会全部丢失

### ISSUE THREAD

**Issue by TonyLiooo** [gshang2017/docker#200]
重启容器后web配置会全部丢失，需要重新进行配置

**Comment by gshang2017** [comment]
QB_TRACKERS_UPDATE_AUTO=false 关了测试下，有可能是更新tracker引起的。

**Comment by TonyLiooo** [comment]
是的,QB_TRACKERS_UPDATE_AUTO=false就不会丢失配置了，请问大佬什么时候能修复下呢，感觉问题还是挺大的，意外重启也会导致配置丢失

**Comment by gshang2017** [comment]
可能是更新tracker时获取的文件格式不对，或者无法正确下载引起的，更改的QB_TRACKERS_LIST_URL需要是ngosang格式，无法正确下载可以在默认配置地址前加上github下载代理（网上有免费的）。

**Comment by TonyLiooo** [comment]
但是我看我使用的tracker list能正确导入进去，链接https://raw.githubusercontent.com/Tunglies/TrackersList/main/all.txt
